domain/domain.go

```diff
@@ -575,6 +575,26 @@ func (do *Domain) UpdateTableStatsLoop(ctx context.Context) error {
 			}
 		}
 	}(do)
+	go func(do *Domain) {
+		id := do.ddl.OwnerManager().ID()
+		cancelCtx, cancelFunc := goctx.WithCancel(goctx.Background())
+		var statsOwner ddl.OwnerManager
+		if do.etcdClient == nil {
+			statsOwner = ddl.NewMockOwnerManager(id, cancelFunc)
+		} else {
+			statsOwner = ddl.NewOwnerManager(do.etcdClient, "stats", id, statistics.StatsOwnerKey, cancelFunc)
+		}
+		statsOwner.CampaignOwners(cancelCtx)
+		for {
+			if statsOwner.IsOwner() {
+				err := do.statsHandle.HandleAutoAnalyze(do.InfoSchema())
+				if err != nil {
+					log.Error(errors.ErrorStack(err))
```

Comment:
Complete the error message. Add the package and function information. You can refer to other logs.